packages/server/src/modules/session/session.model.ts

```diff
@@ -61,6 +61,26 @@ export class Session extends BaseEntity {
     return session ?? null
   }
 
+  public static async findByJWT(token?: string): Promise<Session | null> {
+    if (isNil(token) || token.length < 1) return null
+
+    let data: JWTData | null = null
+
+    try {
+      data = jwt.verify(token, 'scorekeep') as JWTData
+    } catch(err) {
+      if (err.message !== 'jwt malformed') {
```

Comment:
do we not throw any other of the errors that could come from jwt.verify ?

Reply:
We throw if the error is anything but `jwt malformed`